hack/lint-test-cleanup-label.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright the KubeVirt Authors.
+#
+#
+
+EXCLUDE_PATTERN="decorators/decorators.go|\
+tests_suite_test.go|\
+tests/virtctl|\
+tests/network"
```

Comment:
nit: a small concern here is that we'll have a `pkg/something/tests/virtctl`, i.e. that this directory pattern will be found in a place that's not the root of the project.

If we execute grep like this:
```shell
> grep OncePerOrderedCleanup tests/ -rl --include=*.go
tests/decorators/decorators.go
tests/tests_suite_test.go
```

It shows the paths without any prefix, but if we do:
```shell
> grep OncePerOrderedCleanup ./tests/ -rl --include=*.go
./tests/decorators/decorators.go
./tests/tests_suite_test.go
```

Then we get paths stat are prefixed with `./`.
This way, we can change `EXCLUDE_PATTERN` to the following, and make it a bit more precise:
```suggestion
EXCLUDE_PATTERN="./decorators/decorators.go|\
./tests_suite_test.go|\
./tests/virtctl|\
./tests/network"
```